src/views/AddPatientScreen.js

```diff
@@ -17,7 +17,7 @@ const AddPatientScreen = ({ navigation }) => {
         success
         style={styles.button}
       >
-        <Text>Move to patients list</Text>
+        <Text>Move to patients card</Text>
```

Comment:
patient card

Reply:
A to jakieś outdated, chyba wcześniejszy commit 🤔